src/framework/core/js/RemoteModel.js

```diff
@@ -0,0 +1,237 @@
+/*
+Copyright 2017 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_3_0_0 = fluid_3_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    fluid.defaults("fluid.remoteModelComponent", {
```

Comment:
Could do with some substantial comments here explaining what is a "remote model" given clearly this component itself is local : P Perhaps with reference to some diagrams/workflow

Reply:
I've started writing documentation for the [remoteModelComponent](https://github.com/jobara/infusion-docs/blob/FLUID-6209/src/documents/RemoteModelAPI.md). I'll add a link, but it won't be live yet as the documentation needs to be merged in still.